example/genkit_example.dart

```diff
@@ -8,22 +8,24 @@ const baseUrl = 'http://localhost:8080';
 
 void printServerInstructions() {
   print(
-    '-------------------------------------------------------------------\n'
-    '| Before running these examples, make sure the server is running. |\n'
-    '| The server is using a fake model, so does not require API keys. |\n'
-    '| In a separate terminal run:                                     |\n'
-    '|                                                                 |\n'
-    '| \$ cd example/server                                             |\n'
-    '| \$ npm i                                                         |\n'
-    '| \$ npm start                                                     |\n'
-    '-------------------------------------------------------------------\n',
+    r'''
+-------------------------------------------------------------------
+| Before running these examples, make sure the server is running. |
+| The server is using a fake model, so does not require API keys. |
+| In a separate terminal run:                                     |
+|                                                                 |
+| $ cd example/server                                             |
+| $ npm i                                                         |
+| $ npm start                                                     |
+-------------------------------------------------------------------
+''',
   );
 }
 
 // A simple flow that takes a string and returns a string.
 Future<void> _runStringFlow() async {
   print('--- String to String flow ---');
-  final echoStringFlow = defineRemoteAction(
+  final echoStringFlow = defineRemoteAction<String, String>(
```

Comment:
these are actually inferred... at least the response... from `fromResponse` method. No?